website/server/libs/analyticsService.js

```diff
@@ -271,11 +267,28 @@ let _sendPurchaseDataToGoogle = (data) => {
   });
 };
 
+let _setOnce = (data) => {
+  return new Promise((resolve, reject) => {
```

Comment:
LGTM, just one thing here, it looks like `amplitude.identify` already returns a promise, why is it wrapped in another promise?

Reply:
oh, I was just following the patterns we had elsewhere--let me try it without the additional wrapper

Reply:
Yeah that wrapper is needed when the method is using callbacks but in this case it's already returning a promise